pkg/restapi/internal/common/http/common.go

```diff
@@ -35,7 +34,9 @@ func WriteErrorResponse(rw http.ResponseWriter, status int, msg string) {
 }
 
 // WriteResponse writes interface value to response
-func WriteResponse(rw io.Writer, v interface{}) {
+func WriteResponse(rw http.ResponseWriter, v interface{}) {
+	rw.Header().Add("Content-Type", "application/json")
```

Comment:
Can we validate the `Content-Type` in bdd-tests ?

Reply:
yes sure